src/template.html

```diff
@@ -1,29 +1,30 @@
 <!DOCTYPE html>
 <html lang="en">
+  <head>
```

Comment:
Please add the `start` script to the `package.json` file and use `webpack` to start the local server. Your app does not start due to this error.
![error](https://user-images.githubusercontent.com/35504149/165807832-7d6a173f-8b1e-4a40-bb29-0ccd7f280396.png)